lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -637,7 +637,7 @@ def _section_send_email(course, access):
     course_modes = []
     from verified_track_content.models import VerifiedTrackCohortedCourse
     if not VerifiedTrackCohortedCourse.is_verified_track_cohort_enabled(course_key):
-        course_modes = CourseMode.modes_for_course(course_key)
+        course_modes = CourseMode.modes_for_course(course_key, include_expired=True, only_selectable=False)
```

Comment:
The effective definition of `only_selectable` is laid out here: https://github.com/edx/edx-platform/blob/3614eb280031373d8bc9692530562540690e752d/common/djangoapps/course_modes/models.py#L300-L305

Basically, if that parameter is True, `credit` modes are excluded.